gtk2: gnome-themes-extra fails to build with FD.o 19.08beta

The `gnome-themes-extra` fails to find `intltool-update` and thus doesn't build with the new FD.o 19.08 runtime.

Builder log:
```
========================================================================
Building module gnome-themes-extra in /srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/build/gnome-themes-extra-1
========================================================================
]2;flatpak-builder: Building gnome-themes-extraFB: Running: tar xf /srv/buildbot/worker/downloads/downloads/7c4ba0bff001f06d8983cfc105adaac42df1d1267a2591798a780bac557a5819/gnome-themes-extra-3.28.tar.xz --no-same-owner --strip-components=1 -J
FB: Running: flatpak build --die-with-parent --nofilesystem=host --filesystem=/srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/build/gnome-themes-extra-1 --env=SOURCE_DATE_EPOCH=1562238821 '--env=CFLAGS=-O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection' '--env=CXXFLAGS=-O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection' '--env=LDFLAGS=-L/app/lib -Wl,-z,relro,-z,now -Wl,--as-needed' --env=CCACHE_DIR=/run/ccache/disabled --env=PATH=/app/bin:/usr/bin --env=LD_LIBRARY_PATH=/app/lib --env=PKG_CONFIG_PATH=/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig --unshare=network /srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/rofiles/rofiles-Wvm4ji /bin/sh -c 'sed -i '\''s/icons//'\'' themes/HighContrast/Makefile.am'
FB: Running: flatpak build --die-with-parent --env=FLATPAK_BUILDER_BUILDDIR=/run/build/gnome-themes-extra --nofilesystem=host --filesystem=/srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/build/gnome-themes-extra-1 --bind-mount=/run/build/gnome-themes-extra=/srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/build/gnome-themes-extra-1 --build-dir=/run/build/gnome-themes-extra --bind-mount=/run/ccache=/srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/ccache --unshare=network --env=SOURCE_DATE_EPOCH=1562238821 '--env=CFLAGS=-O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection' '--env=CXXFLAGS=-O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection' '--env=LDFLAGS=-L/app/lib -Wl,-z,relro,-z,now -Wl,--as-needed' --env=CCACHE_DIR=/run/ccache/disabled --env=PATH=/app/bin:/usr/bin --env=LD_LIBRARY_PATH=/app/lib --env=PKG_CONFIG_PATH=/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig --env=FLATPAK_BUILDER_N_JOBS=48 --env=NOCONFIGURE=1 /srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/rofiles/rofiles-Wvm4ji ./autogen.sh
libtoolize: putting auxiliary files in '.'.
libtoolize: linking file './ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: linking file 'm4/libtool.m4'
libtoolize: linking file 'm4/ltoptions.m4'
libtoolize: linking file 'm4/ltsugar.m4'
libtoolize: linking file 'm4/ltversion.m4'
libtoolize: linking file 'm4/lt~obsolete.m4'
configure.ac:20: installing './compile'
configure.ac:9: installing './missing'
themes/Adwaita/gtk-2.0/Makefile.am: installing './depcomp'
FB: Running: flatpak build --die-with-parent --env=FLATPAK_BUILDER_BUILDDIR=/run/build/gnome-themes-extra --nofilesystem=host --filesystem=/srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/build/gnome-themes-extra-1 --bind-mount=/run/build/gnome-themes-extra=/srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/build/gnome-themes-extra-1 --build-dir=/run/build/gnome-themes-extra --bind-mount=/run/ccache=/srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/ccache --unshare=network --env=SOURCE_DATE_EPOCH=1562238821 '--env=CFLAGS=-O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection' '--env=CXXFLAGS=-O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection' '--env=LDFLAGS=-L/app/lib -Wl,-z,relro,-z,now -Wl,--as-needed' --env=CCACHE_DIR=/run/ccache/disabled --env=PATH=/app/bin:/usr/bin --env=LD_LIBRARY_PATH=/app/lib --env=PKG_CONFIG_PATH=/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig --env=FLATPAK_BUILDER_N_JOBS=48 /srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/rofiles/rofiles-Wvm4ji ./configure --prefix=/app --disable-dependency-tracking --disable-gtk3-engine
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
...
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no
checking whether NLS is requested... yes
./configure: line 9904: intltool-update: command not found
checking for intltool >= 0.40.0...  found
configure: error: Your intltool is too old.  You need intltool 0.40.0 or later.
]2;flatpak-builder: Cleanup gnome-themes-extraError: module gnome-themes-extra: Child process exited with code 1
```

--- a/gtk2/gtk2.json
+++ b/gtk2/gtk2.json
@@ -103,6 +103,7 @@
           "sha256": "4cf1e5ca4b067a3bed3cdfa658d49ac597d817b2de627a1095214565f862d034"
         }
       ]
-    }
+    },
+    "../intltool/intltool-0.51.json"
   ]
 }

--- a/intltool/intltool-0.51.json
+++ b/intltool/intltool-0.51.json
@@ -1,0 +1,11 @@
+{
+  "name": "intltool",
+  "cleanup": [ "*" ],
+  "sources": [
+    {
+      "type": "archive",
+      "url": "https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-0.51.0.tar.gz",
+      "sha256": "67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd"
+    }
+  ]
+}

